File: Python/programmers/etc/level1/이상한_문자_만들기.py
def solution(s):
    answer = []
    # 주의사항: split(" ")이 아닌 split()은 공백을 전부 없애기 때문에 조건에 맞지 않음
    for st in s.split(" "):
        tmp = ""
        for i in range(len(st)):
            if ~i&1:
                tmp += st[i].upper()
            else:
                tmp += st[i].lower()
        answer.append(tmp)
    return " ".join(answer)


s = "try hello world"
print(solution(s)) # "TrY HeLlO WoRlD"

File: Java/programmers/etc/level1/이상한문자만들기.java
package CodeTest.Java.programmers.etc.level1;

public class 이상한문자만들기 {
    public String solution(String s) {
        String answer = "";
        int cnt = 0;
        String[] arr = s.split("");

        for (String a : arr) {
            cnt = a.equals(" ") ? 0 : cnt+1;
            answer += cnt%2 == 0 ? a.toLowerCase() : a.toUpperCase();
        }
        
        return answer;
    }
}
